cmd/describe/cluster/cmd.go

```diff
@@ -136,16 +136,17 @@ func run(_ *cobra.Command, argv []string) {
 
 	// Print short cluster description:
 	fmt.Printf(""+
-		"Name:        %s\n"+
-		"ID:          %s\n"+
-		"External ID: %s\n"+
-		"AWS Account: %s\n"+
-		"API URL:     %s\n"+
-		"Console URL: %s\n"+
-		"Nodes:       Master: %d, Infra: %d, Compute: %d\n"+
-		"Region:      %s\n"+
-		"State:       %s\n"+
-		"Created:     %s\n",
+		"Name:          %s\n"+
+		"ID:            %s\n"+
+		"External ID:   %s\n"+
+		"AWS Account:   %s\n"+
+		"API URL:       %s\n"+
+		"Console URL:   %s\n"+
+		"Nodes:         Master: %d, Infra: %d, Compute: %d\n"+
+		"Region:        %s\n"+
+		"State:         %s\n"+
+		"Channel Group: %s\n"+
```

Comment:
As per offline conversation, do we want to change this to `Channel:`?

Reply:
cc @jeremyeder 

In our API (and everywhere else) its call `channel_group` b/c its not **exactly** what OCP calls channel (which AFAIK is a link between "versions" in the upgrade graph) but rather an orthogonal concept loosely similar that was invented to partition versions.